VERSION.cmake

```diff
@@ -17,7 +17,10 @@ set( MIRALL_VERSION "${MIRALL_VERSION_MAJOR}.${MIRALL_VERSION_MINOR}.${MIRALL_VE
 # Version with Build ID. Used in the installer
 set( MIRALL_VERSION_FULL "${MIRALL_VERSION}.${MIRALL_VERSION_BUILD}" )
 
-set( MIRALL_VERSION_STRING "${MIRALL_VERSION}${MIRALL_VERSION_SUFFIX}" )
+set( MIRALL_VERSION_STRING "${MIRALL_VERSION_FULL}" )
```

Comment:
I'd like to see a comment that mentions that we use this string for packaging, too. This way, changing or removing the variable won't result in unexpected problems within our CI infra.

Reply:
This variable doesn't quite work, since below the build number is appended another time. We either need to remove that or introduce another variable to either store the value that is assigned after this `if` clause separately for packaging, or by adding another variable that contains the build number in brackets. I think since we encode the build number into the version string already, we can just remove lines 25 ff.